README.md

```diff
@@ -58,9 +99,6 @@ deployAs:
         - key: CTP_PROJECT_KEY
           description: Commercetools project key
           required: true
```

Comment:
You missed to update line number 91 and 92, Please check and correct the module names in this file.
It still says, `used by full ingestion and incremental updater modules`

Reply:
good catch
https://github.com/commercetools/connect-payment-integration-adyen/pull/76/commits/491229dfad08babb346fe79a5bad76c9d08f9a73